resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -58,7 +58,17 @@
                         @if(isset($tickets))
                             @foreach($tickets as $t)
                                 <tr>
-                                    <td><center><a href="/dashboard/controllers/ticket/{{ $t->id }}">{{ $t->date }}</a></center></td>
+                                    <td>
+                                        <center>
+                                            <a href="/dashboard/controllers/ticket/{{ $t->id }}">{{ $t->date }}</a>
+                                            <br />
+                                            @if($t->student_comments)
+                                                <span class="badge badge-danger">Awaiting Your Comments</span>
+                                            @else
+                                                <span class="badge badge-success">Comments Submitted</span>
+                                            @endif
+                                        </center>
+                                    </td>
```

Comment:
To whoever reviews this... I thought this would be a good idea to get more students to add comments, but I am not really sure with it. Please let me know your thoughts

Reply:
Agreed - this is great, just need to get the correct badge to display!